src/components/UserProfileModal.vue

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import { CancelIcon } from '@/assets/icons';
+import { useUserProfileModalStore } from '@/stores/userProfileModal';
+import { useProfileStore } from '@/stores/profile';
+import { storeToRefs } from 'pinia';
+import PositionSmallBadge from '@/components/PositionSmallBadge.vue';
+
+const userProfileModalStore = useUserProfileModalStore();
+const { userProfileModal, userInfo, isLoading, error } = storeToRefs(userProfileModalStore);
+
+const profileStore = useProfileStore();
+const { DEFAULT_PROFILE_IMAGE_URL } = profileStore;
+
+const closeUserProfile = () => {
+  userProfileModalStore.setUserProfileModal(false);
+};
+
+const handleUserPageClick = () => {
+  closeUserProfile();
+};
+</script>
+
+<template>
+  <div
+    v-if="userProfileModal"
+    @click="closeUserProfile"
+    class="flex justify-center items-center z-40 w-full h-full fixed top-0 left-0"
+    style="background-color: rgba(0, 0, 0, 0.5)"
```

Comment:
```suggestion
    class="flex justify-center items-center z-40 w-full h-full fixed top-0 left-0 bg-black/50 backdrop-blur-sm"
```

배경색에 투명도를 넣는 건 이렇게 할 수 있어요!
그리고 모달에 공통적으로 backdrop-blur-sm가 들어가고 있어서 여기도 넣어야 할 것 같습니당

Reply:
앗 그리고 모달이니까 z-index도 50으로 올려주세용!
아무래도 모달들에 공통으로 적용시킬 스타일 클래스를 하나 만들어야겠군요🤔